Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -0,0 +1,41 @@
+Enable Hubble
+==============
+
+Hubble is a fully distributed networking and security observability platform
+for cloud native workloads. It is built on top of Cilium and eBPF to enable
+deep visibility into the communication and behavior of services as well as the
+networking infrastructure in a completely transparent manner.
+
+Deploy Hubble using Helm:
+
+.. parsed-literal::
+
+   helm upgrade cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set global.hubble.enabled=true \\
+      --set global.hubble.listenAddress=":4244" \\
+      --set global.hubble.metricsServer=":54321" \\
+      --set global.hubble.metrics.enabled="{dns,drop,tcp,flow,port-distribution,icmp,http}" \\
+      --set global.hubble.relay.enabled=true \\
+      --set global.hubble.ui.enabled=true
+
+Restart the Cilium daemonset to allow Cilium agent to pick up the ConfigMap changes:
+
+.. parsed-literal::
+
+    kubectl rollout restart -n kube-system ds/cilium
+
+To pick one Cilium instance and validate that Hubble server is properly configured to listen on
+a UNIX domain socket:
+
+.. parsed-literal::
+
+    kubectl exec -n kube-system -t ds/cilium -- hubble observe
+
+To validate that Hubble UI is properly configured, set up a port forwarding for ``hubble-ui`` service:
+
```

Comment:
i also wanted to add a command to validate hubble-relay, but `hubble observe --server hubble-relay:80` doesn't work from cilium pod because of its dns policy: https://cilium.slack.com/archives/C1MATJ5U5/p1591223752145300

Reply:
https://github.com/cilium/cilium/pull/11881